english/change-change/change-change

```diff
@@ -0,0 +1 @@
+-All of a sudden everything stops, and you find yourself in the middle of a clothing store, where all of the merchandise is made with coins. You look down and realize that your clothes are made of coins as well. You decide it's time to change the change on you, and gear up for your next adventure. Time to change change.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.